integrations/openai/src/index.ts

```diff
@@ -194,19 +194,20 @@ export default new bp.Integration({
   register: async () => {},
   unregister: async () => {},
   actions: {
-    generateContent: async ({ input, logger, metadata }) => {
-      const output = await llm.openai.generateContent<LanguageModelId>(
-        <llm.GenerateContentInput>input,
-        openAIClient,
-        logger,
-        {
-          provider,
-          models: languageModels,
-          defaultModel: DEFAULT_LANGUAGE_MODEL_ID,
-        }
-      )
-      metadata.setCost(output.botpress.cost)
-      return output
+    generateContent: async ({}) => {
+      throw new UpstreamProviderError('Fake upstream error')
```

Comment:
leftover ?

Reply:
yes sorry, addressed it [here](https://github.com/botpress/botpress/pull/13533/commits/c4b8702cbcba855686cc717e1228de65cd337fa3) and [here](https://github.com/botpress/botpress/pull/13533/commits/878325bf89e70c0e29988224561525a2f6049693)